resources/scss/site.scss

```diff
@@ -1117,9 +1117,11 @@ span.inline-code {
   margin-top: 0.5em;
 }
 
-#select-platform .display-flex, #select-tags .display-flex {
+#select-platform .display-flex, #select-tags .display-flex, #select-branch {
```

Comment:
If you don't mind, remove the `.display-flex` from the other two selectors as well, I didn't realize when I made them that they were unnecessary.